pkg/controller/datavolume-controller_test.go

```diff
@@ -133,6 +134,33 @@ var _ = Describe("Datavolume controller reconcile loop", func() {
 		Expect(dv.Status.Phase).To(Equal(cdiv1.Pending))
 	})
 
+	It("Should follow the restarts of the PVC", func() {
+		reconciler = createDatavolumeReconciler(newImportDataVolume("test-dv"))
+		_, err := reconciler.Reconcile(reconcile.Request{NamespacedName: types.NamespacedName{Name: "test-dv", Namespace: metav1.NamespaceDefault}})
+		Expect(err).ToNot(HaveOccurred())
+		pvc := &corev1.PersistentVolumeClaim{}
+		err = reconciler.Client.Get(context.TODO(), types.NamespacedName{Name: "test-dv", Namespace: metav1.NamespaceDefault}, pvc)
+		Expect(err).ToNot(HaveOccurred())
+		Expect(pvc.Name).To(Equal("test-dv"))
+
+		dv := &cdiv1.DataVolume{}
+		err = reconciler.Client.Get(context.TODO(), types.NamespacedName{Name: "test-dv", Namespace: metav1.NamespaceDefault}, dv)
+		Expect(err).ToNot(HaveOccurred())
+		Expect(dv.Status.RestartCount).To(BeEquivalentTo(0))
```

Comment:
BeEquivalenTo is used when comparing two types that are derived from each other but are not the same. You will that often with string and another type. If you are comparing the same types, use Equals.

Reply:
right, thanks